wpimath/BUILD.bazel

```diff
@@ -187,6 +199,7 @@ wpilib_cc_shared_library(
         ":shared/wpimath",
         "//wpiutil:shared/wpiutil",
     ],
+    use_debug_name = False,
```

Comment:
Not obvious to me why a bunch of these have `use_debug_name = False`?

Reply:
I attempted to document this in the debug file review.  Holler if it isn't enough.